src/FileList/FileExtInfoLoader.cc

```diff
@@ -1175,56 +1176,65 @@ void FileExtInfoLoader::GetCoordNames(std::string& ctype1, std::string& ctype2,
+            casacore::Vector<casacore::String> axis_names = coord_system.worldAxisNames();
             casacore::Vector<double> pixels(direction_axes.size(), 0);
             casacore::Vector<double> world(direction_axes.size(), 0);
-            casacore::Vector<double> ra_range_vec(2);
-            casacore::Vector<double> dec_range_vec(2);
+            casacore::Vector<double> x_range_vec(2);
+            casacore::Vector<double> y_range_vec(2);
             casacore::String units;
 
             // Get start world coord
             direction_coord.toWorld(world, pixels);
-            ra_range_vec[0] = world[0];
-            std::string ra_start = direction_coord.format(units, casacore::Coordinate::DEFAULT, world[0], 0, true, true);
-            dec_range_vec[0] = world[1];
-            std::string dec_start = direction_coord.format(units, casacore::Coordinate::DEFAULT, world[1], 1, true, true);
+            x_range_vec[0] = world[0];
+            std::string x_start = direction_coord.format(units, casacore::Coordinate::DEFAULT, world[0], 0, true, true);
+            y_range_vec[0] = world[1];
+            std::string y_start = direction_coord.format(units, casacore::Coordinate::DEFAULT, world[1], 1, true, true);
 
             // Get end world coord
             for (unsigned int i = 0; i < pixels.size(); ++i) {
-                pixels[i] = shape[direction_axes[i]] - 1;
+                pixels[i] = image_shape[direction_axes[i]] - 1;
             }
             direction_coord.toWorld(world, pixels);
-            ra_range_vec[1] = world[0];
-            std::string ra_end = direction_coord.format(units, casacore::Coordinate::DEFAULT, world[0], 0, true, true);
-            dec_range_vec[1] = world[1];
-            std::string dec_end = direction_coord.format(units, casacore::Coordinate::DEFAULT, world[1], 1, true, true);
-
-            auto* ra_entry = extended_info.add_computed_entries();
-            ra_entry->set_name("Right Ascension Range");
-            ra_entry->set_value(fmt::format("[{}, {}]", ra_start, ra_end));
-            ra_entry->set_entry_type(CARTA::EntryType::STRING);
-
-            auto* dec_entry = extended_info.add_computed_entries();
-            dec_entry->set_name("Declination Range");
-            dec_entry->set_value(fmt::format("[{}, {}]", dec_start, dec_end));
-            dec_entry->set_entry_type(CARTA::EntryType::STRING);
+            x_range_vec[1] = world[0];
+            std::string x_end = direction_coord.format(units, casacore::Coordinate::DEFAULT, world[0], 0, true, true);
+            y_range_vec[1] = world[1];
+            std::string y_end = direction_coord.format(units, casacore::Coordinate::DEFAULT, world[1], 1, true, true);
+
+            // Set x and y coordinate names
+            if (axis_names(0) == "Right Ascension") {
+                axis_names(0) = "RA";
+            } else if (axis_names(0) == "Longitude") {
+                axis_names(0) = "GLON";
```

Comment:
just LON and LAT. 

G means galactic. There is also ecliptic coordinate as ELON and ELAT. So let just try to display them as LON and LAT for simplicity.